process-dataset/index.js

```diff
@@ -1,90 +1,120 @@
 const fsPromises = require('fs').promises;
+const {
+    firestore
+} = require('../firebase/setup-firebase');
 
-const uploadDatasetAndManifest = require("./steps/upload-dataset-and-manifest");
```

Comment:
Most of the code deleted here has just been moved to process-dataset/process-single-dataset.js